hrjobcontract/tests/phpunit/api/v3/HRJobContractTest.php

```diff
@@ -64,6 +64,16 @@ public function testOnGetContractsWithDetailsInPeriodTheStartDateOnlyAcceptsTheE
 
   /**
    * @expectedException CiviCRM_API3_Exception
+   * @expectedExceptionMessage The contact_id parameter only supports the IN operator
+   *
+   * @dataProvider invalidGetContractsWithDetailsInPeriodContactIDOperator
+   */
+  public function testOnGetContractsWithDetailsInPeriodTheContactIDOnlyAcceptsINOperator($operator) {
```

Comment:
Since none of the other tests here have it, I think it would be better to remove the "On" at the beginning of this test name.